bin/remove-frontend

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env php
+<?php
+
+use Symfony\Component\Filesystem\Filesystem;
+
+require dirname(__DIR__).'/vendor/autoload.php';
+
+$filesystem = new Filesystem();
+$projectDir = dirname(__DIR__);
+
+// Remove routes
+$filesystem->remove($projectDir.'/config/routes/frontend');
+
+// redirect homepage to admin dashboard
+$data = <<<EOM
+app_frontend_homepage:
+    path: /
+    defaults:
+        _controller: FrameworkBundle:Redirect:redirect
+        route: app_backend_dashboard
+
+EOM;
+
+file_put_contents($projectDir.'/config/routes/frontend.yaml', $data);
```

Comment:
why don't you use some functions from Symfony FileSystem, if you have it already instantiated. eg: dump() or appendToFile() ?

Reply:
https://symfony.com/doc/current/components/filesystem.html#dumpfile

Reply:
Just don't know this part. But I replace totally the content (no append option).
I can try dumpfile.